src/blockrelay/compactblock.cpp

```diff
@@ -190,7 +190,7 @@ bool CompactBlock::process(CNode *pfrom, uint64_t nSizeCompactBlock)
     // Because the list of shorttxids is not complete (missing the prefilled transaction hashes), we need
     // to first create the full list of compactblock shortid hashes, in proper order.
     //
-    // Also, create the mapMissingTx from all the supplied tx's in the compact block
+    // Also, create the mapMissingCompactBlockTx from all the supplied tx's in the compact block
```

Comment:
does this mean that current `dev` is using the same data structure to store missing transactions for all propagation types we support?